docs/sql/configuration.md

```diff
@@ -35,70 +35,70 @@ Below is a list of all available settings.
 
 |                   name                   |                                                                       description                                                                       | input_type |      default_value       |
 |------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|------------|--------------------------|
-| Calendar                                 | The current calendar                                                                                                                                    | VARCHAR    | System (locale) calendar |
```

Comment:
Please update the generator for this change

Reply:
Good point, thx. I dropped this last commit from the PR.